### ISSUE TITLE
AnimationGroupPlayer getPosition() always zero

### ISSUE THREAD

**Issue by soonwait** [angular/angular#23785]
https://github.com/angular/angular/blob/d0ccf5f1697c1c0d4e3c8ea7a25f3c87a239df7e/packages/animations/src/players/animation_group_player.ts#L127

```
    function () {
        var /** @type {?} */ min = 0;
        this.players.forEach(function (player) {
            var /** @type {?} */ p = player.getPosition();
            min = Math.min(p, min);
        });
        return min;
    };
```

IS THIS A BUG ???

when min = 0, it always 0 unless player.getPosition() < 0

maybe like this :  `var min = 1`

**Comment by literalpie** [comment]
I'm also seeing this issue. In addition, `player.totalTime` is always 0.

[stackblitz](https://stackblitz.com/edit/angular-getposition-bug?file=src/app/app.component.ts)

**Comment by literalpie** [comment]
related to #18891, but I guess it's different because this is a group, while that issue is about animationPlayer

**Comment by JoostK** [comment]
Hi folks, the fix for this issue (#39983) has landed in 11.0.4, thanks to @literalpie.
